Docs/Configuration.tex

```diff
@@ -2998,66 +3018,68 @@ \subsection{Boot Properties}\label{miscbootprops}
     \item Entry is system (e.g. \texttt{Reset NVRAM}).
   \end{itemize}
 
-  To see all entries picker menu needs to be reloaded in extended mode by pressing
-  \texttt{Spacebar} key. Hiding auxiliary entries may increase boot performance
-  for multidisk systems.
-
+  To display all entries, the picker menu can be reloaded into ``Extended Mode'' by pressing the
+  \texttt{Spacebar} key. Hiding auxiliary entries may increase boot performance on multidisk systems.
 
 \item
   \texttt{LauncherOption}\\
   \textbf{Type}: \texttt{plist\ string}\\
   \textbf{Failsafe}: \texttt{Disabled}\\
-  \textbf{Description}: Register the launcher option in firmware preferences for persistence.
+  \textbf{Description}: Register the launcher option in the firmware preferences for persistence.
 
   Valid values:
 
   \begin{itemize}
   \tightlist
   \item \texttt{Disabled} --- do nothing.
-  \item \texttt{Full} --- create or update top-priority
-  boot option in UEFI variable storage at bootloader startup. For this option
-  to work \texttt{RequestBootVarRouting} is required to be enabled.
+  \item \texttt{Full} --- create or update the top priority
+  boot option in UEFI variable storage at bootloader startup.
+  \begin{itemize}
+  \tightlist
+    \item Enable \texttt{RequestBootVarRouting} for this option to work.
```

Comment:
Should not we keep passive here for consistency?

Reply:
Aim is to go active across the board:
- Set to true to do XYZ
- Enable this for that to happen
- You should do this
- You must not do that
etc

Reply:
Hrm. I do not believe I saw stuff like "Enable X for Y" in the specs.

Reply:
So what would you prefer?

Reply:
Found "For this option to work, \texttt{RequestBootVarRouting} is required to be enabled."